phpBB/phpbb/controller/helper.php

```diff
@@ -142,6 +142,12 @@ public function route($route, array $params = array(), $is_amp = true, $session_
 
 		$base_url = $context->getBaseUrl();
 
+		// Regarding PHPBB3-13493 - if we request url ending on / there is no $page_name
+		if (strpos($base_url, $page_name) === false)
```

Comment:
Replace with either:
`if (!$page_name || strpos($base_url, $page_name) === false)`
or
`if ($page_name && strpos($base_url, $page_name) === false)`

and the tests will pass.
I just don't know which one of the conditions is the right one.